doc/docs/1.12.x/deploy-manage/deploy/prometheus/prometheus.md

```diff
@@ -0,0 +1,88 @@
+!!! note 
+    To monitor a Pachyderm cluster
+    with Prometheus, 
+    a ***Pachyderm Enterprise License*** is required. 
+
+# Monitor your Pachyderm cluster with Prometheus
```

Comment:
We should indicate that this is one way to deploy Prometheus but we really have no preference and don't want to be responsible for the deployment of Prometheus.

Reply:
Also, above the first point, we should call out that pachd and the pipeline pods expose Prometheus metrics